vendor/github.com/HabanaAI/gohlml/hlml.go

```diff
@@ -0,0 +1,695 @@
+/*
```

Comment:
Is there a reason why we have all these go libraries and packages in the Kepler code base?